admin/common_auth/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label=(u'User Name'), required=True)
```

Comment:
Correct. The kwarg name has to be username. so `authenticate(username=email, password=password)` will work. 

I'm 99% sure you could do `authenticate(email, password)`  and everything would be fine. The first example is more explicit that you're shoving email into the username field.

IIRC keyword args in python are just like regular arguments except you have the option to ignore position. So if you wanted to be mean you could do `authenticate(password=password, username=email)` but you couldn't do `authenticate(password, email)`.